tardis/energy_input/gamma_ray_channel.py

```diff
@@ -3,13 +3,57 @@
 import pandas as pd
 import astropy.units as u
 import radioactivedecay as rd
+from tardis.model.matter.decay import IsotopicMassFraction
 
 from tardis.energy_input.util import KEV2ERG
 
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO)
 
 
+def evolve_mass_fraction(
+    raw_isotope_mass_fraction,
+    time_delta,
+    model_isotope_time,
+):
+    """
+    Function to evolve the isotope abundance over time.
+
+    Parameters
+    ----------
+    raw_isotope_mass_fraction : pd.DataFrame
+        mass fractions of isotopes.
+    time_delta : np.array
```

Comment:
Is this actually a time delta? We have to be careful to pick the correct time to measure the decay at. I would suggest the "effective time" array, which is the central time of each time bin.